### ISSUE TITLE
Build failure with metabang-bind HEAD

### ISSUE THREAD

**Issue by ilovezfs** [dimitri/pgloader#492]
Attempting to build pgloader 3.3.2 with metabang-bind-20161204-git failed, whereas the build succeeded with metabang-bind-20141106-git.

Bisecting the problem it looks like pgloader has been unable to build with metabang-bind ever since https://github.com/gwkkwg/metabang-bind/commit/e82928f0b2a32c526ccb1edd4804a4310f7bd9b0. The prior commit (https://github.com/gwkkwg/metabang-bind/commit/ee35be8042e2ba7f3f734f70458236b4f0547d62) is fine, and is probably the same as metabang-bind-20141106-git.

The error looks like this:
```
==> make pgloader-standalone BUILDAPP=buildapp
buildapp    --require sb-posix --require sb-bsd-sockets --require sb-rotate-byte                        \
                       --sbcl sbcl                            \
                       --load-system pgloader               \
                       --load src/hooks.lisp                   \
                       --entry pgloader:main                   \
                       --dynamic-space-size 4096         \
                       --compress-core                    \
                       --output build/bin/pgloader
;; loading system "pgloader"
;; loading file #P"/private/tmp/pgloader-20161212-29891-17lun2j/pgloader-3.3.2/src/hooks.lisp"
While evaluating the form starting at line 29, column 0
  of #P"/private/tmp/pgloader-20161212-29891-17lun2j/pgloader-3.3.2/dumper-2SKVI5f7.lisp":

debugger invoked on a SB-KERNEL:SIMPLE-PACKAGE-ERROR in thread
#<THREAD "main thread" RUNNING {1001BBF0B3}>:
  The name NIL does not designate any package.

Type HELP for debugger help, or (SB-EXT:EXIT) to exit from SBCL.

restarts (invokable by number or by possibly-abbreviated name):
  0: [RETRY   ] Retry EVAL of current toplevel form.
  1: [CONTINUE] Ignore error and continue loading file "/private/tmp/pgloader-20161212-29891-17lun2j/pgloader-3.3.2/dumper-2SKVI5f7.lisp".
  2: [ABORT   ] Abort loading file "/private/tmp/pgloader-20161212-29891-17lun2j/pgloader-3.3.2/dumper-2SKVI5f7.lisp".
  3:            Ignore runtime option --load "/private/tmp/pgloader-20161212-29891-17lun2j/pgloader-3.3.2/dumper-2SKVI5f7.lisp".
  4:            Skip rest of --eval and --load options.
  5:            Skip to toplevel READ/EVAL/PRINT loop.
  6: [EXIT    ] Exit SBCL (calling #'EXIT, killing the process).

(PACKAGE-NAME NIL)
```

**Comment by scymtym** [comment]
This is most likely an SBCL bug only present in certain builds of the 1.3.12 release. Downgrading to 1.3.11, upgrading to master, or building 1.3.13 with the immobile-code feature disabled will probably solve the issue.

**Comment by ilovezfs** [comment]
Yep this is fixed now that sbcl 1.3.13 has shipped. Thanks for the help!
